tests/unit/models/test_utils.py

```diff
@@ -2,12 +2,80 @@
 
 from xrpl.models.currencies import IssuedCurrency
 from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.requests import AccountInfo
+from xrpl.models.transactions import AccountSet, AccountSetAsfFlag, Payment, PaymentFlag
+from xrpl.models.utils import is_kw_only_attr_defined_in_dataclass
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_FEE = "0.00001"
+_SEQUENCE = 19048
+_DOMAIN = "asjcsodafsaid0f9asdfasdf"
+_TRANSACTION = AccountSet(
+    account=_ACCOUNT,
+    fee=_FEE,
+    set_flag=AccountSetAsfFlag.ASF_DISALLOW_XRP,
+    domain=_DOMAIN,
+    sequence=_SEQUENCE,
+)
+
+_SEED = "randomsecretseedforakey"
 
 currency = "BTC"
 issuer = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
 
+_DESTINATION = "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
+_XRP_AMOUNT = "10000"
+
 
 class TestUtils(TestCase):
     def test_kwargs_req(self):
-        with self.assertRaises(XRPLModelException):
-            IssuedCurrency(currency, issuer)
+        if is_kw_only_attr_defined_in_dataclass():
+            with self.assertRaises(XRPLModelException):
+                IssuedCurrency(currency, issuer)
+        else:
+            with self.assertRaises(TypeError):
+                IssuedCurrency(currency, issuer)
```

Comment:
I recommend putting this if-else block in a reusable context manager to avoid needing it in every test/avoid repetition.

Reply:
What do you mean by a re-usable context manager? Do you mean something like:
```
def testWrapper(pythonVersionDiscriminator, oldVersion, newVersion):
    if pythonVersionDiscriminator():
        <some steps in the unit tests>
        <raise XRPLModelException>
    else:
        <some more steps>
        <raise TypeError>
```
?

Reply:
https://book.pythontips.com/en/latest/context_managers.html